SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2582,8 +2596,11 @@ void USpatialReceiver::OnAsyncPackageLoaded(const FName& PackageName, UPackage*
 			CriticalSectionSaveState CriticalSectionState(*this);
 
 			EntityWaitingForAsyncLoad AsyncLoadEntity = EntitiesWaitingForAsyncLoad.FindAndRemoveChecked(Entity);
-			PendingAddActors.Add(Entity);
-			PendingAddComponents = MoveTemp(AsyncLoadEntity.InitialPendingAddComponents);
+			if (HasEntityBeenRequestedForDelete(Entity))
```

Comment:
This seems a little bit weird - shouldn't this be negated?
But I am not very familiar with the async loading code.

Reply:
I agree - this looks incorrect

Reply:
I'm not actually sure we need to do anything here - this should only happen when receiving an actor across the wire, which means this wasn't the server that created it in the first place. So perhaps we can just remove this check.

Reply:
hm, yes, good point - delete?

Reply:
Agreed; I can't think of a case where you might finish async loading after creating and deleting the actor.